desktop/libs/aws/src/aws/s3/s3_test.py

```diff
@@ -64,12 +64,11 @@ def test_is_root():
 def test_s3datetime_to_timestamp():
   f = s3.s3datetime_to_timestamp
   eq_(1424983327, f('Thu, 26 Feb 2015 20:42:07 GMT'))
-  eq_(1424983327, f('2015-02-26T20:42:07.000Z'))
+  eq_(1424983327, f('2015-02-26T20:42:07.040Z'))
```

Comment:
Would it work to keep the old test line too?

Reply:
Previous test validate exactly '.000Z' because it's was in assert rule(`datetime[-5:] == '.000Z'`) and 
any other numbers should raise exception.
But now there is only check for  'Z' and '.'.
I think there is no reason for old test case.

Reply:
Always good to test multiple potential corner cases, will add it back

Reply:
Ok, added.